src/js/src/components/SearchBox.vue

```diff
@@ -209,31 +211,25 @@ export default {
     selectSearchMethod(selected) {
       console.log(selected)
       if(selected === 'imgSearch') {
-        this.futureImgSearch = !this.futureImgSearch
-        this.futureImgSearch === true ? this.futureUrlSearch = false : null
-        //this.solrSettings.urlSearch ? this.updateSolrSettingUrlSearch(!this.solrSettings.urlSearch) : null
-        //this.updateSolrSettingImgSearch(!this.solrSettings.imgSearch)
+        this.updateFutureSolrSettingImgSearch(!this.futureSolrSettings.imgSearch)
+        this.futureSolrSettings.imgSearch === true ? this.updateFutureSolrSettingUrlSearch(false) : null
       }
       else if(selected === 'urlSearch') {
-        this.futureUrlSearch = !this.futureUrlSearch
-        this.futureUrlSearch === true ? this.futureImgSearch = false : null
-        //this.solrSettings.imgSearch ? this.updateSolrSettingImgSearch(!this.solrSettings.imgSearch) : null
-        //this.updateSolrSettingUrlSearch(!this.solrSettings.urlSearch)
+        this.updateFutureSolrSettingUrlSearch(!this.futureSolrSettings.urlSearch)
```

Comment:
See comment above

Reply:
This is intended behavior. The selected checkbox should simply flip to the opposite of that it is now. So i'll leave this as it is.

Reply:
👍